doc/source/upstream-bug-fixes.rst

```diff
@@ -4,645 +4,651 @@
+|:Upstream bug: :mysqlbug:`53588` - Blackhole : Specified key was too long; max key length is 1000 bytes      |
+|:JIRA bug: :psbug:`1126`                                                                                     |
+|:Upstream state: Verified (checked on 2018-05-24)                                                            |
+|:Fix Released: :rn:`5.5.59-38.11`                                                                            |
+|:Upstream fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`83264` - uint3korr should stop reading four instead of three bytes on x86          |
-|:Launchpad bug: :bug:`1628417`                                                                               |
+|:JIRA bug: :psbug:`3567`                                                                                     |
 |:Upstream state: Closed                                                                                      |
 |:Fix Released: :rn:`5.5.52-38.3`                                                                             |
 |:Upstream fix: 5.5.57                                                                                        |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`68052` - SSL Certificate Subject ALT Names with IPs not respected with --ssl-ver...|
-|:Launchpad bug: :bug:`1673656`                                                                               |
-|:Upstream state: Verified (checked on 2017-08-11)                                                            |
+|:JIRA bug: :psbug:`1076`                                                                                     |
+|:Upstream state: Verified (checked on 2018-05-24)                                                            |
 |:Fix Released: :rn:`5.5.57-38.9`                                                                             |
 |:Upstream fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`83814` - Add support for OpenSSL 1.1                                               |
-|:Launchpad bug: :bug:`1702903`                                                                               |
-|:Upstream state: Verified (checked on 2017-08-11)                                                            |
+|:JIRA bug: :psbug:`1105`                                                                                     |
+|:Upstream state: Closed                                                                                      |
```

Comment:
Please avoid whitespace at the end of the line for "Upstream state" too

Reply:
Same answer on highlighted trailing whitespaces (just a diff visual highlighting; the whole table is filled with spaces)